DB/src/main/java/io/deephaven/db/util/NoLanguageDeephavenSession.java

```diff
@@ -80,9 +80,10 @@ public boolean hasVariableName(String name) {
     }
 
     @Override
-    public void setVariable(String name, Object value) {
-        super.setVariable(name, value);
-        variables.put(name, value);
+    public void setVariable(String name, @Nullable Object newValue) {
+        Object oldValue = getVariable(name, null);
```

Comment:
ditto this with @Nullable in getVariable, esp since it delegates to getVariable with no `newValue`, but will throw, so the param isn't "optional if null" as one might suspect...